CMakeLists.txt

```diff
@@ -34,8 +34,10 @@ find_package(serialization_utils REQUIRED)
 find_package(Threads)
 find_package(rt)
 
-# export de dependencies
-ament_export_dependencies(pybind11 Eigen3 real_time_tools time_series
+include(cmake/find_boost_components.cmake)
```

Comment:
You mean it could be converted to something like `FindBoostIostream.cmake` which is then used via `find_package(BoostIostream)`?

Reply:
I quickly looked into this.  I think providing a proper `FindBoostIostream.cmake` file would add a lot of overhead as it would need to forward optional arguments and define various `BoostIostream_XXX` variables.  So I feel like it's a bit too much effort for the gain.

I'll keep it as is for now to not block this PR but I'm happy to further discuss and potentially refactor later.